api/s2n.h

```diff
@@ -446,7 +446,7 @@ extern struct s2n_cert_chain_and_key *s2n_connection_get_selected_cert(struct s2
  * @param cert_length This return value represents the length of the s2n certificate chain `chain_and_key`.
  */
 S2N_API
-extern int s2n_get_cert_chain_length(const struct s2n_cert_chain_and_key *chain_and_key, uint32_t *cert_length);
+extern int s2n_cert_get_cert_chain_length(const struct s2n_cert_chain_and_key *chain_and_key, uint32_t *cert_length);
```

Comment:
If we're naming after the structure, this actually needs to be "s2n_cert_chain_get_length" (or maybe "s2n_cert_chain_and_key_get_chain_length", but idk if the "key" part is necessary for this context). And "s2n_cert_get_cert_from_cert_chain" -> "s2n_cert_chain_get_cert" (or "s2n_cert_chain_and_key_get_cert")

Reply:
Yes, we could keep the context, how about 

```c
int s2n_cert_get_der(const struct s2n_cert *cert, const uint8_t **out_cert_der,
                                    uint32_t *cert_length);
int s2n_cert_chain_and_key_get_length (const struct s2n_cert_chain_and_key *chain_and_key, 
                                    uint32_t *cert_length);
int s2n_cert_chain_and_key_get_cert(const struct s2n_cert_chain_and_key *chain_and_key, 
                                    struct s2n_cert **out_cert, const uint32_t cert_idx);
```